webhook/user_info.go

```diff
@@ -55,15 +55,13 @@ func (m *UserInfoMutator) Handle(ctx context.Context, req admission.Request) adm
 		annotations[k] = v
 	}
 
-	marshaledUserInfo, err := json.Marshal(req.UserInfo)
+	dis.Annotations = annotations
+
+	err := dis.SetUserInfo(req.UserInfo)
 	if err != nil {
-		m.Log.Errorw("error encoding UserInfo", "error", err)
+		m.Log.Errorw("error defining UserInfo", "error", err, "name", dis.Name, "namespace", dis.Namespace)
```

Comment:
small nit: disruptionName and disruptionNamespace?

Reply:
Good catch! Definitely worth changing that to keep the same everywhere + avoid collision with this kind of generic tag names.

Reply:
It's not consistent with other log usage in the chaos controller 😬 
I'm fine aligning this TBH, just wanted to mention we changed this due to metrics tagging, not logs tagging.

If we want to be consistent everywhere I can adapt accordingly 🤷 

Not sure if the alignment would risk to break anything (saved view or log filtering by disruption name?) for us I don't think so.

WDYT?

Reply:
IMO we should use disruptionName and disruptionNamespace everywhere, and make it consistent. We could do this in a separate PR where we review all our tags instead if you think it has not its place here?

Reply:
I would at least prefer to have the opportunity to discuss this with @ptnapoleon because he did the first round of tags alignment.

I'm fine to either decision we make, I just want to be sure we first agree on which convention to choose 😅